dhctl/pkg/app/ssh.go

```diff
@@ -154,6 +155,13 @@ func DefineBecomeFlags(cmd *kingpin.CmdClause) {
 		Envar(configEnvName("ASK_BECOME_PASS")).
 		Short('K').
 		BoolVar(&AskBecomePass)
+
+	cmd.PreAction(func(c *kingpin.ParseContext) error {
+		if SSHUser == "root" {
+			AskBecomePass = false
+		}
+		return nil
```

Comment:
I think we need to fail with error instead. Because for root user this flag should not set

Reply:
Agreed, made a new commit